dnf.spec

```diff
@@ -187,19 +190,35 @@ Systemd units that can periodically download package upgrades and apply them.
 %prep
 %autosetup
 
+%if (0%{?fedora} && 0%{?fedora} <= 32) || 0%{?rhel}
+    mkdir build-py3
+%endif
 
 %build
 
-%global _vpath_builddir build-py3
-%cmake -DPYTHON_DESIRED:FILEPATH=%{__python3} -DDNF_VERSION=%{version}
-%cmake_build
-%cmake_build --target doc-man
-
+%if (0%{?fedora} && 0%{?fedora} <= 32) || 0%{?rhel}
```

Comment:
And I would drop all these ifs, keeping the `if` branch and dropping the `else`. In other words, remove the new macros and revert back to the old ones.

If @Conan-Kudo succeeds in removing the old macros in RHEL 9 (:grin:), we'll have the same situation across all our spec files and we can apply the same fix everywhere. (Neal if you think otherwise feel free to comment, IMO it's marginal and I'm just trying to make the code short and avoid branching if not necessary)

Reply:
I agree. This will make the specfiles much more readble and in sync.
I'm committing new changes and testing now how it builds for fedoras and rhel.